server/adaptors/integrations/__data__/repository/otel-services/schemas/traces-1.0.0.mapping.json

```diff
@@ -0,0 +1,201 @@
+          "ignore_above": 128,
+          "type": "keyword"
+        },
+        "startTime": {
+          "type": "date_nanos"
+        },
+        "endTime": {
+          "type": "date_nanos"
+        },
+        "droppedAttributesCount": {
+          "type": "long"
+        },
+        "droppedEventsCount": {
+          "type": "long"
+        },
+        "droppedLinksCount": {
+          "type": "long"
+        },
+        "status": {
+          "properties": {
+            "code": {
+              "type": "long"
+            },
+            "message": {
+              "ignore_above": 128,
+              "type": "keyword"
+            }
+          }
+        },
+        "attributes": {
+          "type": "object",
+          "data_stream": {
+            "properties": {
+                "dataset": {
+                  "ignore_above": 128,
+                  "type": "keyword"
+                },
+                "namespace": {
+                  "ignore_above": 128,
+                  "type": "keyword"
+                },
+                "type": {
+                  "ignore_above": 56,
+                  "type": "keyword"
+                }
+              }
+            }
+          }
+        },
+        "events": {
+          "type": "nested",
+          "properties": {
+            "name": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            },
+            "@timestamp": {
+              "type": "date_nanos"
+            },
+            "observedTimestamp": {
+              "type": "date_nanos"
+            }
+          }
+        },
+        "links": {
+          "type": "nested",
+          "properties": {
+            "traceId": {
+              "ignore_above": 256,
+              "type": "keyword"
+            },
+            "spanId": {
+              "ignore_above": 256,
+              "type": "keyword"
+            },
+            "traceState": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256
+                }
+              }
+            }
+          }
+        },
+        "instrumentationScope": {
+          "properties": {
+            "name": {
+              "type": "keyword"
+            },
+            "version": {
+              "type": "keyword"
+            },
+            "droppedAttributesCount": {
+              "type": "integer"
+            },
+            "schemaUrl": {
+              "type": "keyword"
+            }
+          }
+        },
+        "schemaUrl": {
+          "type": "keyword"
+        }
+      }
+    }
+  },
+  "composed_of": [],
+  "version": 1,
+  "_meta": {
+    "description": "Observability Traces Mapping Template",
+    "catalog": "observability",
+    "type": "traces",
+    "correlations": [
+      {
+        "field": "serviceName",
+        "foreign-schema": "services",
+        "foreign-field": "serviceName"
+      }
+    ]
+  }
+}
```

Comment:
I think there is a formatting error here.

Reply:
Thanks